cells.gemspec

```diff
@@ -12,6 +12,13 @@ Gem::Specification.new do |spec|
   spec.description = 'View Models for Ruby and Rails, replacing helpers and partials while giving you a clean view architecture with proper encapsulation.'
   spec.license     = 'MIT'
 
+  spec.metadata['bug_tracker_uri'] = "#{spec.homepage}/issues"
+  spec.metadata['changelog_uri'] = "#{spec.homepage}/blob/HEAD/CHANGES.md"
+  spec.metadata['documentation_uri'] = "https://www.rubydoc.info/gems/cells/#{spec.version}"
```

Comment:
This is the correct URL: https://trailblazer.to/2.1/docs/cells

Reply:
Fixed in [`75bbcc0`](https://github.com/trailblazer/cells/pull/501/commits/75bbcc0906ed7059029928134ac60bb64174a24b).